sound/soc/sof/intel/hda-compress.c

```diff
@@ -25,7 +25,7 @@ int hda_probe_compr_assign(struct snd_sof_dev *sdev,
 {
 	struct hdac_ext_stream *stream;
 
-	stream = hda_dsp_stream_get(sdev, cstream->direction);
+	stream = hda_dsp_stream_get(sdev, cstream->direction, false);
```

Comment:
comment on commit message: 'exclude... from disabling'. can we reword as 'enable'?

Reply:
I also find the 'entry disabled' wording odd, it's really the mode that is disabled so there is no transition.